### PR TITLE
Use Eth Recover

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
@WoeOm Please check the spec

File: frame/bridge/ethereum/backing/src/lib.rs [hunk 1]
```diff
@@ -736,9 +736,26 @@ impl<T: Trait> Sign<BlockNumber<T>> for Module<T> {
...
-			crypto::secp256k1_ecdsa_recover(signature, &hashing::blake2_256(message.as_ref()))
-		{
+		fn eth_signable_message(message: &[u8]) -> Vec<u8> {
+			let mut l = message.len();
+			let mut rev = Vec::new();
+
+			while l > 0 {
+				rev.push(b'0' + (l % 10) as u8);
+				l /= 10;
+			}
```
Comment (HackFisher) [inline frame/bridge/ethereum/backing/src/lib.rs:746]: What's this?

Comment (WoeOm) [reply frame/bridge/ethereum/backing/src/lib.rs:746]: May be related to https://docs.ethers.io/ethers.js/v5-beta/cookbook-signing.html

Comment (AurevoirXavier) [reply frame/bridge/ethereum/backing/src/lib.rs:746]: > What's this?

Same things, in darwinia-claims.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**